GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -93,8 +93,8 @@ extension AssetsCatalogParser {
 // MARK: - ACTool
 
 extension AssetsCatalogParser {
-  fileprivate func loadAssetCatalog(at path: String) -> [[String: Any]]? {
-    let command = Command("xcrun", arguments: "actool", "--print-contents", path)
+  fileprivate func loadAssetCatalog(at path: Path) -> [[String: Any]]? {
+    let command = Command("xcrun", arguments: "actool", "--print-contents", String(describing: path))
```

Comment:
I think `path.description` feels more like a property on `Path` while `String(describing: path)`, although doing the same thing, feels more like a conversion or builder to me.